plugins/woocommerce/src/Admin/Features/MarketingRecommendations/Init.php

```diff
@@ -54,21 +56,60 @@ public static function get_specs() {
 		return $specs;
 	}
 
+	/**
+	 * Process specs.
+	 *
+	 * @param array|null $specs Marketing recommendations spec array.
+	 * @return array
+	 */
+	protected static function evaluate_specs( array $specs = null ) {
```

Comment:
The purpose of having this is to bring MarketingRecommendations way of processing specs closer to what we have with the rest of the RemoteSpecsEngine. In the near future, we may be able to standardize all RemoteSpecsEngine for ease of maintenance.